### PR TITLE
Update to Slimefun RC-27

### PR DESCRIPTION
**Pull request by md5sha256**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -94,20 +94,16 @@
...
 
         <dependency>
-            <groupId>com.github.TheBusyBiscuit</groupId>
-            <artifactId>CS-CoreLib</artifactId>
-            <version>6e17183656</version>
+            <groupId>io.github.baked-libs</groupId>
+            <artifactId>dough-api</artifactId>
+            <version>1.0.2</version>
             <scope>provided</scope>
             <exclusions>
```
Comment (TheBusyBiscuit) [inline pom.xml:106]: CS-CoreLib 1 is no longer in use.
This should be removed...

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (md5sha256) [reply pom.xml:106]: I don't follow... Didn't i remove it?

Comment (TheBusyBiscuit) [reply pom.xml:106]: Nope, you replaced it with dough.
Dough is not a plugin, makes no sense to have a dependency on it this way, it is shaded into Slimefun.

Comment (md5sha256) [reply pom.xml:106]: Ah yes, i was wondering about why cscl was there in the first place.

Comment (TheBusyBiscuit) [reply pom.xml:106]: Because this wasn't updated in a long time :LUL:

Comment (md5sha256) [reply pom.xml]: Aye. Should be fixed now

**Review by TheBusyBiscuit — APPROVED** [review]

**Pull request merged**